spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -535,15 +535,14 @@ def fixture_record(fixture_name)
       end
     end
   end
-  describe 'location facet values for Recap items' do
-    it 'marquand recap items have a location value of marquand and recap' do
-      expect(@added_title_246['location_display']).to eq ['Remote Storage: Marquand Library use only']
-      expect(@added_title_246['location']).to eq ['ReCAP']
-    end
-    it 'non-rare recap items only have a location value of recap' do
-      expect(@online_at_library['location_display']).to include 'Mudd Off-Site Storage: Contact mudd@princeton.edu'
-      expect(@online_at_library['location']).to include 'ReCAP'
-      expect(@online_at_library['location']).not_to include 'Mudd Manuscript Library'
+  describe 'location facet values for Recap remote storage items' do
+    it 'marquand recap items have a location value of marquand' do
```

Comment:
rephrase description to: location facet values for Recap items

Reply:
rephrase to: marquand recap items display as marquand and are in the marquand location facet

Reply:
The rephrase review is from @mzelesky